go.mod

```diff
@@ -1,69 +1,70 @@
 module jester.noble.xyz
 
-go 1.22.7
+go 1.24
+
+toolchain go1.24.0
```

Comment:
Normally we can get away with not having the `toolchain` here, but could be wrong!

```suggestion
```

Reply:
You're right, probably not necessary here. Although one nice feature is that if a user doesn't have 1.24 installed and they try to build from source without having 1.24, go will automatically download the correct version. 
Regardless, I'm ok with removing it.

Reply:
Oh I didn't know that was a benefit, I guess we can add it back later?